lib/referee-sinon.test.js

```diff
@@ -872,4 +872,14 @@ describe("referee-sinon", function() {
             assert.match(pass.args[0][0], "Expectation met");
         });
     });
+
+    describe("match from referee with sinon assert", function() {
```

Comment:
It might be worthwhile referencing this PR, so people can find the motivation for this test